lib/logstash/inputs/lumberjack.rb

```diff
@@ -48,7 +51,7 @@ def register
 
     @logger.info("Starting lumberjack input listener", :address => "#{@host}:#{@port}")
     @lumberjack = Lumberjack::Server.new(:address => @host, :port => @port,
```

Comment:
We should do some extra validation -

1) If ssl is true, require both ssl_certificate and ssl_key
2) If ssl is false, reject the presence of ssl_certificate and ssl_key, or at least warn that they are not used in this configuration.